lib/kwtool_web/templates/keyword/index.html.eex

```diff
@@ -16,8 +18,8 @@
           <td><%= keyword.phrase %></td>
           <td><%= keyword.inserted_at %></td>
           <td><span class="status-badge--added"><%= keyword.status %></span></td>
-          <td>
-            <span><%= link "View", class: "button-detail", to: Routes.keyword_path(@conn, :show, keyword) %></span>
+          <td class="keyword-list__action-column">
+            <span><%= link "View", class: "btn btn-sm btn-secondary", to: Routes.keyword_path(@conn, :show, keyword) %></span>
```

Comment:
@hanam1ni agreed with your comment on this button, I updated it to use the bootstrap class instead.